### PR TITLE
NIFI-3793 - switch to use project.version

### PR DESCRIPTION
**Pull request by YolandaMDavis**
Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [ ] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [ ] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-assembly?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-assembly?
- [ ] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: nifi-toolkit/nifi-toolkit-admin/pom.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@ language governing permissions and limitations under the License. -->
         <dependency>
             <groupId>org.apache.nifi</groupId>
             <artifactId>nifi-client-dto</artifactId>
-            <version>${client.version}</version>
+            <version>${project.version}</version>
```
Comment (joewitt) [inline nifi-toolkit/nifi-toolkit-admin/pom.xml]: I would recommend avoiding these variables here and instead just use '1.2.0-SNAPSHOT'.  The former has proven problematic during the release process and the latter has proven highly reliable and gets automatically updated by the versions plugin during the release process.

File: nifi-toolkit/nifi-toolkit-admin/pom.xml [hunk 2]
```diff
@@ -57,7 +57,7 @@ language governing permissions and limitations under the License. -->
         <dependency>
             <groupId>org.apache.nifi</groupId>
             <artifactId>nifi-web-security</artifactId>
-            <version>${client.version}</version>
+            <version>${project.version}</version>
```
Comment (joewitt) [inline nifi-toolkit/nifi-toolkit-admin/pom.xml]: I would recommend avoiding these variables here and instead just use '1.2.0-SNAPSHOT'.  The former has proven problematic during the release process and the latter has proven highly reliable and gets automatically updated by the versions plugin during the release process.

**Comment by YolandaMDavis** [comment]
@joewitt thanks switched to refer to 1.2.0-SNAPSHOT

**Comment by bbende** [comment]
Thanks Yolanda, will review...

**Pull request closed without merging**